views/group/pagination.pug

```diff
@@ -0,0 +1,31 @@
+div.has-margin-top-4.has-margin-bottom-3
+  nav.pagination(role="pagination", aria-label="pagination").is-centered.is-small
+    if pOpt.pageNum <= 5
+      ul.pagination-list
+        - for (i = 0; i < pOpt.pageNum; i++) {
+          li 
+            if i == pOpt.current
+              a(href=`/groups?page=${i}&limit=${pOpt.limit}`).pagination-link.is-current #{i + 1}
```

Comment:
You don't have to use interpolation here, just use pug's `=` operator like this:

```a(href=`/groups?page=${i}&limit=${pOpt.limit}`).pagination-link.is-current= i + 1```

This applies to line no. 10, 21, 23, 26 and 31